lizmap/modules/lizmap/controllers/edition.classic.php

```diff
@@ -424,8 +424,12 @@ private function getDataFields($datasource){
         // If postgresql, get real geometryType from geometry_columns (jelix prop gives 'geometry')
         if( $this->provider == 'postgres' and $this->geometryType == 'geometry' ){
           $cnx = jDb::getConnection($this->layerId);
-          $res = $cnx->query('SELECT type FROM geometry_columns WHERE f_table_name = '.$cnx->quote($this->tableName));
-          $res = $res->fetch();
+          $sql = "SELECT type FROM geometry_columns";
+          $sql.= " WHERE 2>1";
+          $sql.= " AND t_table_schema = " . $cnx->quote($schema);
```

Comment:
Can you change `t_table_schema` to `f_table_schema` ?

Reply:
sorry. Done.